### PR TITLE
version resolved to use data in Marker for flutter_map

### PR DESCRIPTION
**Pull request by sandeshgyw**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_compass: ^0.6.1
-  flutter_map: ^0.13.1
+  flutter_map: 
+    git: 
+      url: "https://github.com/sandeshgyw/flutter_map.git"
```
Comment (xni06) [inline pubspec.yaml:16]: Could you achieve the same result my using the released version of `flutter_map_location` but override the `flutter_map` using `depencency_override` in your app's `pubspec.yaml`?

**Comment by xni06** [comment]
@sandeshgyw it is safe to close/cancel this PR as it has been resolved another way - thanks.

File: pubspec.yaml [hunk 2]
```diff
@@ -11,7 +11,10 @@ dependencies:
...
+      ref: "13db571"
```
Comment (Xennis) [inline pubspec.yaml:17]: Please don't add unpublished Dart packages here. Let's use the official flutter_map.

You can't publish a Dart packages build on top of unpublished repos/packages.

File: example/ios/Podfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Uncomment this line to define a global platform for your project
```
Comment (Xennis) [inline example/ios/Podfile:1]: Dear @sandeshgyw can you please provide some information on what that codes here is based?

**Review by Xennis — CHANGES_REQUESTED** [review]
See comments above

**Pull request closed without merging**